### PR TITLE
Reformat with scalafmt

### PR DESCRIPTION
**Pull request by jvican**
Same as https://github.com/sbt/sbt/pull/3125.

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -56,6 +56,13 @@ lazy val lm = (project in file("librarymanagement")).
       val sdirs = (managedSourceDirectories in Compile).value
       val base = baseDirectory.value
       (((srcs --- sdirs --- base) pair (relativeTo(sdirs) | relativeTo(base) | flat)) toSeq)
+    },
+    commands += Command.command("scalafmtCheck") { state =>
```
Comment (dwijnand) [inline build.sbt]: you need to add this to the root (lmRoot)

**Review by dwijnand — CHANGES_REQUESTED** [review]

File: build.sbt (hunk 1 above)
Comment (jvican) [reply build.sbt]: Done.

**Comment by jvican** [comment]
I have updated the formatting of all the sources here. Right now, we strictly follow what upstream sbt does. Do you mind having a look @eed3si9n ?

**Review by eed3si9n — APPROVED** [review]

**Comment by eed3si9n** [comment]
LGTM

**Comment by jvican** [comment]
Since this is just a port of sbt/sbt reformatting, there's already a signoff and CI is green, can we merge this and I rebase https://github.com/sbt/librarymanagement/pull/88?

**Pull request merged**